### PR TITLE
Code refactoring (constants for templates, subjects ..)

### PR DESCRIPTION
**Pull request by shyyka**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-XXX)

## Code reviewers

- [ ] @yurii-kokhalevych 
- [ ] @AlbaMontis
- [ ] @avoytovich

### Second Level Review

- [ ] @lhalam
- [ ] @4asof4ik

### CODE REVIEW

**Review by avoytovich — DISMISSED** [review]

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by avoytovich — DISMISSED** [review]

File: server/controllers/guest.controller.js [hunk 1]
```diff
@@ -41,20 +41,20 @@ module.exports = {
           const {first_name: firstName, last_name: lastName} = guest.User.Profile || '';
           const route = guest.User.is_invited && '/signup' || '/';
 
-          mailer({
+          let data = Object.assign(invitation, {
```
Comment (AlbaMontis) [inline server/controllers/guest.controller.js]: const

File: server/views/emails/activated/html.hbs [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <h2 class="center">Congratulation</h2>
   <h3 class="center">Welcome to Da-Mi!</h3>
   <blockquote>
-  <h1>your account was activated</h1> 
+  <h1>your account was activated</h1>
```
Comment (AlbaMontis) [inline server/views/emails/activated/html.hbs]: Your account was activated.

File: server/views/emails/activated/html.hbs [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
   <p>to enjoy application follow this<a href={{url}}> link </a>. </p>
```
Comment (AlbaMontis) [inline server/views/emails/activated/html.hbs]: To enjoy application please follow...

File: server/controllers/support.controller.js [hunk 4]
```diff
@@ -1,18 +1,17 @@
...
   support(req, res) {
     let assignSupport = Object.assign({}, req.body);
-    let data = {
-      subject: messages.problem,
-      img: 'error.png',
-      email: 'dami.suppor@gmail.com',
-      firstname: req.body.name,
-      lastname: req.body.surname,
+
+    let data = Object.assign(support, {
```
Comment (AlbaMontis) [inline server/controllers/support.controller.js]: > Use const by default, unless a variable needs to be reassigned. 
https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations

File: server/controllers/guest.controller.js [hunk 5]
```diff
@@ -41,20 +41,20 @@ module.exports = {
...
             host: constants.URL,
             route: route,
-            subject: 'Invitation',
-            firstname: firstName,
-            lastname: lastName,
+            firstName: firstName,
```
Comment (AlbaMontis) [inline server/controllers/guest.controller.js]: Just `firstName`.

File: server/controllers/guest.controller.js [hunk 6]
```diff
@@ -41,20 +41,20 @@ module.exports = {
...
+            lastName: lastName,
```
Comment (AlbaMontis) [inline server/controllers/guest.controller.js]: The same as above.

File: server/controllers/guest.controller.js [hunk 7]
```diff
@@ -41,20 +41,20 @@ module.exports = {
           const {first_name: firstName, last_name: lastName} = guest.User.Profile || '';
           const route = guest.User.is_invited && '/signup' || '/';
 
-          mailer({
+          let data = Object.assign(invitation, {
             host: constants.URL,
             route: route,
```
Comment (AlbaMontis) [inline server/controllers/guest.controller.js]: Just `route`.

Comment (shyyka) [reply server/controllers/guest.controller.js]: Done

File: server/controllers/guest.controller.js (hunk 6 above)
Comment (shyyka) [reply server/controllers/guest.controller.js]: Done

File: server/controllers/guest.controller.js (hunk 1 above)
Comment (shyyka) [reply server/controllers/guest.controller.js]: Done

File: server/views/emails/activated/html.hbs (hunk 2 above)
Comment (shyyka) [reply server/views/emails/activated/html.hbs]: Done

File: server/views/emails/activated/html.hbs (hunk 3 above)
Comment (shyyka) [reply server/views/emails/activated/html.hbs]: Done

File: server/controllers/guest.controller.js (hunk 5 above)
Comment (shyyka) [reply server/controllers/guest.controller.js]: Done

**Review by avoytovich — DISMISSED** [review]

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by 4asof4ik — APPROVED** [review]

**Review by yurii-kokhalevych — DISMISSED** [review]

**Review by yurii-kokhalevych — APPROVED** [review]

**Review by AlbaMontis — APPROVED** [review]

**Pull request merged**